tests/framework/PostProcessors/InterfacedPostProcessor/gold/hs_sync_all/synced.csv

```diff
@@ -1,6 +1,6 @@
-y,x,filename
-1.5,0.0,synced_0.csv
-1.11270166538,0.0,synced_1.csv
-1.88729833462,0.0,synced_2.csv
-1.5,-0.774596669241,synced_3.csv
-1.5,0.774596669241,synced_4.csv
+x,y,filename
+0.0,1.5,synced_0.csv
+-0.774596669241,1.5,synced_1.csv
+0.774596669241,1.5,synced_2.csv
+0.0,1.11270166538,synced_3.csv
+0.0,1.88729833462,synced_4.csv
```

Comment:
The "synced" files should be following the order given by the DataSet, so the sorting of the ROMs shouldn't have changed this, right? The order of the outputs should be controlled by the DataSet that they're put in. If not, we probably need to figure out why. Did changing the line in the Sobol control the variable order?

Reply:
Based on my reading of test_historySetSyncAll.xml, the dataset is generated in the make step, and then synced in the print step, and this changes the dataset at the make step so it is different when it gets to the hs_sync_all/synced.csv  later.    Yes, it controls the variable order (the order of the ROMs).

Reply:
Okay, I will just open an issue to investigate that then. The variables should be entirely in the order of how they're listed in the input file I think. In the meantime, these changes are fine to me.